packages/block-editor/src/hooks/settings.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * WordPress dependencies
+ */
+import { addFilter } from '@wordpress/hooks';
+import { hasBlockSupport } from '@wordpress/blocks';
+
+const hasSettingsSupport = ( blockType ) =>
+	hasBlockSupport( blockType, 'settings', true );
```

Comment:
I'm guessing the default should be false, no?

```suggestion
	hasBlockSupport( blockType, 'settings', false );
```